contrib/imgprune/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+source /usr/local/bin/lib_entrypoint.sh
+
+req_env_var GCPJSON GCPNAME GCPPROJECT IMGNAMES
+
+gcloud_init
+
+# Different purpose from quay.io/libpod/imgts
+EXCLUDE="$IMGNAMES $IMAGE_BUILDER_CACHE_IMAGE_NAME"
+# For safety's sake + limit nr background processes
+PRUNE_LIMIT=10
+THEFUTURE=$(date --date='+1 hour' +%s)
+THRESHOLD=$(date --date='90 days ago' +%s)
+FORMAT='value[quote](name,selfLink,creationTimestamp,labels)'
+PROJRE="/v1/projects/$GCPPROJECT/global/"
+BASE_IMAGE_RE='cloud-base'
+FILTER="selfLink~$PROJRE AND creationTimestamp<$(date --date='30 days ago' --iso-8601=date) AND NOT name=($EXCLUDE)"
+TODELETE="/tmp/todelete"
+
+echo "Searching images for pruning candidates:"
+gcloud compute images list --format="$FORMAT" --filter="$FILTER" | \
+    while read name selfLink creationTimestamp labels
+    do
+        last_used=$(egrep --only-matching --max-count=1 'last-used=[[:digit:]]+' <<< $labels || true)
+
+        if [[ -z "$last_used" ]]
+        then # image pre-dates addition of tracking labels
+            echo "Marking $name for deletion: Missing 'last-used' metadata, labels: '$labels'"
+            echo "$name" >> $TODELETE
+            continue
+        fi
+
+        last_used_timestamp=$(date --date=@$(cut -d= -f2 <<< $last_used || true) +%s || true)
+        if [[ -z "$last_used_timestamp" ]] || [[ "$last_used_timestamp" -ge "$THEFUTURE" ]]
+        then
+            echo "Marking $name for deletion: Missing or invalid last-used timestamp: $last_used_timestamp"
+            echo "$name" >> $TODELETE
+            continue
+        fi
+
+        if [[ "$last_used_timestamp" -le "$THRESHOLD" ]]
+        then
+            echo "Marking $name for deletion: Too old: $last_used_timestamp <= $THRESHOLD"
```

Comment:
As a courtesy to human beings, could you change this to:
```bash
    last_used_ymd=$(date --date=@$last_used_timestamp --iso-8601=date)
    echo "Marking $name for deletion: last used on $last_used_ymd"
```
For ease of debugging, you could amend line 23 to something like
```bash
echo "Searching images for pruning candidates; threshold is $(date --date=@$THRESHOLD ....
```

Reply:
Kk, got something like this in